config/dev.json

```diff
@@ -3,6 +3,7 @@
   "bardRoot": "https://terra-bard-dev.appspot.com",
   "bondUrlRoot": "https://broad-bond-dev.appspot.com",
   "calhounUrlRoot": "https://terra-calhoun-dev.appspot.com",
+  "dataRepoUrlRoot": "https://cors-anywhere.herokuapp.com/https://jade.datarepo-dev.broadinstitute.org",
```

Comment:
This was a hack to get around the CORS issue - it will be removed once Data Repo fixes it

Reply:
Cool, I'm assuming this will be pre-merge